settings.gradle.kts

```diff
@@ -7,6 +7,65 @@ pluginManagement {
    }
 }
 
+@Suppress("UnstableApiUsage")
+dependencyResolutionManagement {
+   repositoriesMode.set(RepositoriesMode.PREFER_SETTINGS)
+
+   repositories {
+      google()
```

Comment:
Do we use something from `google()`?

Reply:
Good question, I removed the Google and Kotlin Wasm repos and the tests passed locally. Let's see if CI passes too...